pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotQueryResource.java

```diff
@@ -101,10 +102,16 @@ public class PinotQueryResource {
   @Inject
   ControllerConf _controllerConf;
 
+  @Context
```

Comment:
Just for my info, how is the injection handled across different requests on various endpoints?